src/pocketmine/lang/locale/tur.ini

```diff
@@ -1,4 +1,5 @@
-
+#Turkish language resource by PainTR
```

Comment:
Git give you attribution. Don't add additional things since you are contributing to *our* software.